tests/gui/components/svg-icon.spec.js

```diff
@@ -72,6 +72,17 @@ describe('Browser: SVGIcon', function () {
       m.chai.expect(element.children().attr('src')).to.equal(imgData)
     })
 
+    it('should use an empty src if there is a parsererror', function () {
+      // The following is invalid, because there's no closing tag for `foreignObject`
+      const iconContents = '<svg><foreignObject></svg>'
+      const imgData = `data:image/svg+xml,`
+      $rootScope.iconContents = iconContents
+
+      const element = $compile('<svg-icon path="iconContents">Resin.io</svg-icon>')($rootScope)
```

Comment:
Here we take advantage of Angular's string variable processing, notice how we put `iconContents` on the `$rootScope`. We don't have to do that here, but it's generally safer to do so because it means we don't have to escape any characters.